lib/auth/user.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2015 Gravitational, Inc.
```

Comment:
FYI, `new_web_user.go` has been renamed to `user.go`. No code changes in regards to `user` functions.